Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py

```diff
@@ -1155,8 +1155,15 @@ def delete_devices():
             'Device Name': device_name,
             'Deletion status': True
         })
+    batch_size = demisto.args().get("batch_size", 20)
+    try:
```

Comment:
why not batch_size.isdigit() ?

Reply:
because that would fail on an int.
I can assign the default get value to be "20" and then it should work, but this seems more robust, don't you think?